src/nuketesting/_cli/main.py

```diff
@@ -145,16 +145,21 @@ def main(
     like pytest does.
 
     """
-    test_run_arguments = CLIRunArguments(
-        nuke_executable=nuke_executable,
-        test_directory=test_path,
-        config=config,
-        run_in_terminal_mode=terminal,
-        pytest_args=pytest_arg,
-        runner_name=runner_name,
-    )
-
-    _run_tests(test_run_arguments)
+    try:
+        test_run_arguments = CLIRunArguments(
+            nuke_executable=nuke_executable,
+            test_directory=test_path,
+            config=config,
+            run_in_terminal_mode=terminal,
+            pytest_args=tuple(pytest_arg),
+            runner_name=runner_name,
+        )
+        _run_tests(test_run_arguments)
+
+    except CLICommandError as e:
+        ctx = click.get_current_context()
```

Comment:
I'd prefer to call this context. It's only a difference of 4 letters and it makes reading the code easier